src/lib/components/environments/enviroments-panel/sub-components/quick-help/QuickHelp.svelte

```diff
@@ -64,8 +64,9 @@
   </p>
   <p><img class="w-100" src={environment} alt="" /></p>
   <p class="heading-para text-textColor">
-    All the global variables are marked with G. Where as all local variables
-    within an environment are marked with E.
+    All the global variables are marked with <span class="global-env-G">G</span>
```

Comment:
move it to const in constants file and import it here

Reply:
Resolved